### ISSUE TITLE
Size of emotes

### ISSUE THREAD

**Issue by Vyxxxx** [An00nymushun/DiscordFreeEmojis#26]
Is there a way to make emotes default send as size 32 instead of 64 or 128 whatever it's sending as?

**Comment by marcussacana** [comment]
Yes, just change in the .plugin.js source, find for: ?size=64, then change to 32... save the plugin and reload the discord.

**Comment by Vyxxxx** [comment]
> Yes, just change in the .plugin.js source, find for: ?size=64, then change to 32... save the plugin and reload the discord.

Thanks! That's what I had planned to do but I was unsure which one to edit, appreciate it!

**Comment by 81322068** [comment]
Any way to change the size in the userscript version?

**Comment by marcussacana** [comment]
https://github.com/An00nymushun/DiscordFreeEmojis/blob/7bb933027ae6bd65c557d141574be390b3c1942e/DiscordFreeEmojis.user.js#L113
I didn't tested, but try change `emoji.url` to `emoji.url + "?size=32"`

**Comment by 81322068** [comment]
Thanks for your help, nope that doesn't work.
```result.content = result.content.replace(`<${emoji.animated ? "a" : ""}:${emoji.originalName || emoji.name}:${emoji.id}>`, emoji.url + "?size=32");```

**Comment by marcussacana** [comment]
Maybe the emoji.url isn't just a url then, you will need put after this like something like console.log(emoji.url); and see the value to we know what can be done after that.

**Comment by 81322068** [comment]
Oh I think I got the issue:.
emoji.url is `https://cdn.discordapp.com/emojis/824467565900201994.png?v=1`
here if I remove `?v=1` and replace it with `?size=32` it works. `https://cdn.discordapp.com/emojis/824467565900201994.png?size=32`

Is there a way to change the code to remove ?v=1 and replace it with ?size=32?

**Comment by Withern10** [comment]
yeah could anyone try to find a way?

**Comment by cyan-2048** [comment]
`+ &size=40`

**Comment by cuanim** [comment]
Thank you cyan,
Had to add Quotations  to make it work tho
`+ "&size=64"`

**Comment by cyan-2048** [comment]
oh shockers i forgot to add quotes sorry for that

**Comment by An00nymushun** [comment]
There is a 64px userscript version, it would be way easier to edit that instead
